core/arch/arm/plat-marvell/conf.mk

```diff
@@ -86,6 +86,78 @@ CFG_NUM_THREADS ?= CFG_TEE_CORE_NB_CORE
 CFG_CORE_HEAP_SIZE ?= 131072
 endif
 
+ifeq ($(PLATFORM_FLAVOR),cn10ka)
+include core/arch/arm/cpu/cortex-armv8-0.mk
+$(call force,CFG_TEE_CORE_NB_CORE,24)
+$(call force,CFG_CLUSTERS_PER_NODE,1)
```

Comment:
Please remove CFG_CLUSTERS_PER_NODE`, it is not used (same below)

Reply:
True. Will remove for cn10k variants.